webapp/src/components/testing/TestSteps.vue

```diff
@@ -142,24 +170,24 @@ const props = defineProps<{
 interface Tested {
   image: List<{
     input: { filename: string; image: ImageData };
-    output: { truth: string; correct: boolean };
+    output: { truth: string; correct: boolean; predicted : number, label : number };
   }>;
   tabular: {
     labels: {
       input: List<string>;
-      output: { truth: string; correct: string };
+      output: { truth: string; correct: string, label : string };
     };
     results: List<{
       input: List<string>;
-      output: { truth: string; correct: boolean };
+      output: { truth: string; correct: boolean; predicted : number, label : number };
```

Comment:
I think I added them in case someone else wants to perform further analysis, it is always usefull to have the string of the category the element was initially, in my opinion. But I can remove it